### DISCUSSION TITLE
Feedback Fritzbox 6430

### DISCUSSION THREAD

**Discussion by Surfer-76** [Freetz-NG/freetz-ng#422] [Allgemein]
Hallo, da die 6430 immernoch als untested steht, wollte ich Rückmeldung geben, dass ich mehrere Images gebaut habe für 2 verschiedne 6430 Boxen und die Fritzbox einwandfrei funktiioniert. Konnte nicht alle Pakete testen, aber...
Artikelnummer 20002779

WLAN OK
Kabel Vodafone Sync OK
LAN OK
VPN OK
Telefon VoiP OK
USB mount OK
bis jetzt nichts was an AVm funktionen nicht funktioniert hätte

freetz Funktionen
telnet, dropbear, crond. python,usb, OK.

Wegen weiteren Informationen melden.

**Comment by fda77** [comment]
Danke für die Rückmeldung, ich habs angepasst: https://github.com/Freetz-NG/freetz-ng/commit/d89d7eba2c4b5d3d426c5abfa5db33b323c64828
Wenn die Box noch bootet und das Webinterface zugreifbar ist dürfte schon das meiste funktionieren. Einzelne Pakete können immer mal Probleme machen, das ist nicht so schlimm.
Mit welchem Program/Script hast du geflasht?
Kannst du noch einen Screenshot von "AVM-Info" anhängen? (einfach die Bilddatei ins Textfenster ziehen)

**Comment by Surfer-76** [comment]
Hallo ich habe es normal über EVA per ftp gemacht nach dem booten, nach der guten Anleitung von hier
https://web.archive.org/web/20190103183930/https://www.ip-forum.eu/howto-aendern-des-branding-und-installieren-der-retail-firmware-bei-fritz-box-cable-160?sid=0846fe3e61522dda9d09ca34fa6b9390

Hier noch die AVM-Info, wie man sieht sind einige Pakete installier
![Unbenannt](https://user-images.githubusercontent.com/95878032/145667427-1b54254d-3d9b-4eac-baf9-953c84fe4c98.PNG)
t

**Comment by fda77** [comment]
Danke, das kommt im meine Sammlung.
Man kann nur auf der Box selbst die CPU auslesen: "Intel(R) Atom(TM) CPU 612 @ 600MHz". Dazu gibt es bei Google nur 1 Treffer: https://pastebin.com/S6GRiVSM Auch bei Boxmatrix wird die noch als 1,2 GHz gelistet: https://boxmatrix.info/wiki/FRITZ!Box_6430_Cable#Overview , falls du hippie im IRC bescheid sagen möchtest, er hat keinen Github-Account.
Flashen sollte auch mit tools/push_firmware funktionieren da die Box das Layout wie die 6460 hat. Mit den Zertifikaten wird da aber gar nichts gemacht. Gibt es zu dem ip-forum.eu Link eine Kurzversion? @PeterPawn hat da doch ein paar angelegt
